index.js

```diff
@@ -4,14 +4,19 @@ const LastCommitLog = require('last-commit-log');
 /**
  * A safe way to query for git infos.
  *
+ * @param {object} logger node logger compatible interface to log errors
  * @return {object|undefined} Returns {hash, gitTag} or undefined in case of an error
  */
-async function getLastCommit() {
+async function getLastCommit(logger) {
   try {
     const lastCommitLog = new LastCommitLog();
     const { hash, gitTag } = await lastCommitLog.getLastCommit();
     return { hash, gitTag };
-  } catch (err) {}
+  } catch (err) {
+    if (logger) {
```

Comment:
On second thought, I'd suggest to just remove this commit.  Users that don't use git probably don't want flooded logger messages.

Reply:
Have you tested with and without git tag?  We shouldn't add a property git tag if it's missing or blank.

Reply:
Will add that later.

Reply:
Thanks again for your efforts, this is incredibly resourceful at a time of need.